File: backEnd/platform-automation/src/main/java/com/mokatest/platform/demos/domain/ui/dto/step/ClosePageStepDTO.java
package com.mokatest.platform.demos.domain.ui.dto.step;// 关闭页面操作

import com.mokatest.platform.demos.domain.ui.uiEnum.browser.ColsePageMode;
import lombok.Data;

@Data
public class ClosePageStepDTO extends StepBaseDTO {

    private String stepType;
    private ColsePageMode closePageMode; // 关闭页面模式
    private Integer customIndex; // 自定义关闭页面索引
}

File: frontEnd/src/types/dto/stepDTO/ClosePageStepDTO.ts
// 关闭页面操作
import {ClosePageMode} from "@/types/enum/browser/ClosePageMode";
import {StepBaseDTO} from "@/types/dto/base/StepBaseDTO";

export class ClosePageStepDTO extends StepBaseDTO {

    stepType: string = "CLOSE_PAGE";
    closePageMode: ClosePageMode | string = "CURRENT"; // 关闭页面模式
    customIndex?: number = 0; // 自定义关闭页面索引
}
